suites/quincy/rgw/tier-2_rgw_test-bucket-notifications-kafka-ssl.yaml

```diff
@@ -80,7 +80,7 @@ tests:
   - test:
       name: notify on multipart upload events with kafka_broker and SSL security
       desc: notify on multipart upload events with kafka_broker and SSL security
-      polarion-id: CEPH-83575407
+      polarion-id: CEPH-83575471
       comments: known issue (bz-2149259)
```

Comment:
@hmaheswa , since this particular suite we are using only in 6.1 metadata file,
and bz-2149259 is fixed in 6.1 could you please remove comments: known issue

Reply:
also we may need to include this comment in pacific suite if testcase is present there..since issue still there in 6.0 build